### PR TITLE
WORKSHOP BUG FIX remove latest.rst, remove latest from index.rst, and add latest.rst to .gitignore so doc build passes

### PR DESCRIPTION
**Pull request by mindyls**
In conjunction with https://github.com/NRLMMD-GEOIPS/geoips/pull/692 , this will ensure command line tests/test_all.sh will pass for geoips_clavrx.

THis is required to fix NRLMMD-GEOIPS/geoips#666

Running locally, it did NOT reinstall geoips_clavrx, and successfully returned 0 (when on branch 666-final-bug-fixes-prior-to-workshop-6  for geoips repo)

```
***************************************************************************
Package geoips_clavrx is already installed!
***************************************************************************
...
Return values:
  Python install:
  sphinx html:    0
  sphinx latex:   None
  latex pdf make: None
  warnings html:
  warnings latex:
  warnings make:

Final retval: 0
```

### CODE REVIEW

**Review by christopher-selman — APPROVED** [review]

**Review by cpcamacho — APPROVED** [review]

**Review by jsolbrig — APPROVED** [review]

File: docs/source/releases/index.rst
Comment (mindyls) [inline docs/source/releases/index.rst:26]: Do not merge this yet.  The *.rst release note check should actually be failing because this is modified and latest.rst was deleted.  Make sure that check is failing before merging.

Comment (mindyls) [reply docs/source/releases/index.rst:26]: image

Comment (mindyls) [reply docs/source/releases/index.rst:26]: The RST modified check successfully failed, and all others passed.  Will override and merge, since we were intentionally chaging the RST files (and we should not have to change them again)

Comment (mindyls) [reply docs/source/releases/index.rst:26]: image

**Comment by mindyls** [comment]
Thanks for getting this worked out!  RST release notes modified successfully failed, everything else passed!

image

**Pull request merged**
